gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/converters/StringToAttributeKindConverter.java

```diff
@@ -69,20 +71,38 @@ public Value doConvert(String value, AttributeKind attributeKind, Value.Builder
         valueBuilder.setValueType(ValueType.STRING_MAP);
         valueBuilder.putAllStringMap(convertToMap(value));
         return valueBuilder.build();
+      case TYPE_STRING_ARRAY:
+        valueBuilder.setValueType(ValueType.STRING_ARRAY);
+        valueBuilder.addAllStringArray(convertToArray(value));
+        return valueBuilder.build();
       default:
         break;
     }
     return null;
   }
 
+  private List<String> convertToArray(String jsonString) {
+    if (StringUtils.isEmpty(jsonString)) {
+      return List.of();
+    }
+
+    try {
+      return objectMapper.readValue(jsonString, LIST_TYPE_REFERENCE);
+    } catch (IOException e) {
+      log.warn("Unable to read List JSON String data from: {}. With error:", jsonString, e);
+      log.warn("Setting data as empty list instead");
```

Comment:
Do we need the second warning? it should be implicit or merge it with the above line itself.

Reply:
Will merge into the other log message.